plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/model/Settings.java

```diff
@@ -85,7 +85,7 @@ public final boolean equals(final Object obj) {
         return this.getIdentity().equals(settings.getIdentity())
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1067)